lib/pbio/drv/usb/usb_common_desc.c

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2025 The Pybricks Authors
+
+// Descriptors which are shared across devices
+// (i.e. WebUSB, Microsoft OS descriptors)
+
+#include <pbdrv/config.h>
+
+#if PBDRV_CONFIG_USB
+
+#include "usb_common_desc.h"
+
+#if PBIO_VERSION_LEVEL_HEX == 0xA
+#define PYBRICKS_CODE_URL   "alpha.pybricks.com"
+#elif PBIO_VERSION_LEVEL_HEX == 0xB
+#define PYBRICKS_CODE_URL   "beta.pybricks.com"
+#else
+#define PYBRICKS_CODE_URL   "code.pybricks.com"
+#endif
+
+const pbdrv_usb_webusb_url_desc_union_t pbdrv_usb_webusb_landing_page = {
+    .s = {
+        .bLength = sizeof(pbdrv_usb_webusb_url_desc_t) + sizeof(PYBRICKS_CODE_URL) - 1,
+        .bDescriptorType = WEBUSB_DESC_TYPE_URL,
+        .bScheme = WEBUSB_URL_SCHEME_HTTPS,
+        .url = PYBRICKS_CODE_URL,
+    },
+};
+
+#define MS_20_REGISTRY_DATA_EXTRA_SZ                    \
+    2 + sizeof(PBDRV_USB_MS_20_REG_PROPERTY_NAME) +     \
+    2 + sizeof(PBDRV_USB_MS_20_REG_PROPERTY_VAL) +      \
+    2
```

Comment:
```suggestion
    2 + sizeof(PBDRV_USB_MS_20_REG_PROPERTY_VAL) + 2
```
To show that this is a single item.

Or even better, add the extra 0 character to `PBDRV_USB_MS_20_REG_PROPERTY_VAL`.